builtin/providers/aws/resource_aws_route_test.go

```diff
@@ -49,6 +49,43 @@ func TestAccAWSRoute_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSRoute_ipv6Support(t *testing.T) {
+	var route ec2.Route
+
+	//aws creates a default route
+	testCheck := func(s *terraform.State) error {
+
+		name := "aws_egress_only_internet_gateway.foo"
+		gwres, ok := s.RootModule().Resources[name]
+		if !ok {
+			return fmt.Errorf("Not found: %s\n", name)
+		}
+
+		if *route.EgressOnlyInternetGatewayId != gwres.Primary.ID {
+			return fmt.Errorf("Egress Only Internet Gateway Id (Expected=%s, Actual=%s)\n", gwres.Primary.ID, *route.EgressOnlyInternetGatewayId)
+		}
+
+		return nil
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
I'm just following the same convention as the rest of the tests - we can address this separately